### PR TITLE
Improve test coverage of Elvis expression

### PR DESCRIPTION
**Pull request by pranavan15**
## Purpose
Improve the test coverage of Elvis expression. 

## Approach
Removed the unnecessary methods and added new test cases.

### CODE REVIEW

File: compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/codegen/CodeGenerator.java [hunk 1]
```diff
@@ -1301,8 +1297,8 @@ public void visit(BLangAwaitExpr awaitExpr) {
         } else {
             valueRegIndex = this.getOperand(-1);
         }
-        genNode(awaitExpr.expr, this.env);
-        Operand futureRegIndex = awaitExpr.expr.regIndex;
+        genNode(awaitExpr.getExpression(), this.env);
```
Comment (SupunS) [inline compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/codegen/CodeGenerator.java]: We don't need to access this via the getter.
The convention we followed throughout is, all the fields in the implemented nodes (i.e: BLang*) are public, and we can directly access it.

Shall we keep those as it is? Applicable for all places..

File: compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/tree/expressions/BLangAwaitExpr.java [hunk 2]
```diff
@@ -49,7 +48,7 @@ public String toString() {
     }
 
     @Override
-    public ExpressionNode getExpression() {
+    public BLangExpression getExpression() {
```
Comment (SupunS) [inline compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/tree/expressions/BLangAwaitExpr.java]: As per the above comment, we don't need this change too

File: compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/codegen/CodeGenerator.java (hunk 1 above)
Comment (pranavan15) [reply compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/codegen/CodeGenerator.java]: Ack; The reason for changing them was to include the getter methods within the test coverage. But as it's **not** something crucial, will revert those changes to align with the usual conventions.

Comment (pranavan15) [reply compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/codegen/CodeGenerator.java]: Reverted with 4545a8d and 50ee9e5

File: compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/tree/expressions/BLangAwaitExpr.java (hunk 2 above)
Comment (pranavan15) [reply compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/tree/expressions/BLangAwaitExpr.java]: Reverted with 4545a8d

**Comment by hasithaa** [comment]
@SupunS Can you review this.

**Pull request merged**